instructions.md

```diff
@@ -0,0 +1,203 @@
+# Build an Appointments App
+
+## Overview
+This project is slightly different than others you have encountered thus far on Codecademy. Instead of a step-by-step tutorial, this project contains a series of open-ended requirements which describe the project you'll be building. There are many possible ways to correctly fulfill all of these requirements, and you should expect to use the internet, Codecademy, and other resources when you encounter a problem that you cannot easily solve.
```

Comment:
Nick and I have been talking about editing this boilerplate overview for all challenge projects. The only change is removing the first sentence.

```suggestion
Instead of a step-by-step tutorial, this project contains a series of open-ended requirements which describe the project you'll be building. There are many possible ways to correctly fulfill all of these requirements, and you should expect to use the internet, Codecademy, and other resources when you encounter a problem that you cannot easily solve.
```